src/delivery-options/DeliveryOptions.vue

```diff
@@ -148,7 +155,7 @@ export default {
       }
 
       const { cc } = this.$configBus.address;
-      const requirements = addressRequirements[addressRequirements.hasOwnProperty(cc) ? cc : 'nl'];
+      const requirements = addressRequirements[addressRequirements.hasOwnProperty(cc) ? cc : NL];
```

Comment:
klopt het dat dit een constante moet zijn?

Reply:
Yes. Is ook "nl"